src/betterproto/plugin/parser.py

```diff
@@ -1,7 +1,9 @@
 import itertools
 import pathlib
 import sys
-from typing import List, Iterator
+from typing import Iterator, List, Tuple, Union
+
+from google.protobuf.descriptor import Descriptor
```

Comment:
if only used for type annoation then this import should be under `if TYPE_CHECKING:`